src/components/DepositWidget/Form.tsx

```diff
@@ -7,31 +7,57 @@ import { TokenBalanceDetails } from 'types'
 import { formatAmountFull, parseAmount } from 'utils'
 import BN from 'bn.js'
 
-const Wrapper = styled.tr`
+const DynamicWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &.fixedContainer {
+    position: fixed;
```

Comment:
Since we are using styled-components:
```js
${props => props.responsive && `
    position: fixed;
    top: 0;
    left: 0;
    bottom: 0;
    right: 0;
    background: #000000b5;
`
}
```
without classes